core/src/main/scala/org/bitcoins/core/util/BytesUtil.scala

```diff
@@ -0,0 +1,53 @@
+package org.bitcoins.core.util
+
+import org.bitcoins.core.number.UInt64
+import org.bitcoins.core.protocol.CompactSizeUInt
+import org.bitcoins.crypto.{CryptoBytesUtil, Factory, NetworkElement}
+import scodec.bits.ByteVector
+
+import scala.annotation.tailrec
+
+trait BytesUtil extends CryptoBytesUtil {
```

Comment:
nit: Just do `has-a` instead of `is-a` with `CryptoBytesUtil`

Reply:
I personally think that for `Util` files it makes sense to use `is-a` as everything is static functions anyway. Also the goal in doing it this way is to allow users to call all the `BytesUtil` methods that the `crypto` project needed as if they were still defined in the `core` project (like before). I.e. this un-breaks our API.